modules/minimint-wallet/src/bitcoind.rs

```diff
@@ -1,9 +1,20 @@
+use std::error::Error;
+
 use crate::Feerate;
 use async_trait::async_trait;
-use bitcoin::{Block, BlockHash, Network, Transaction};
-use bitcoincore_rpc::bitcoincore_rpc_json::EstimateMode;
-use bitcoincore_rpc::RpcApi;
-use tracing::warn;
+use bitcoin::{BlockHash, Transaction};
+use thiserror::Error;
+
+#[cfg(feature = "native")]
```

Comment:
Feature gating is good for now, should think about splitting off `minimint-wallet-api` though.

Reply:
cleaned it up in 79f3589e62dada86c1a35b3dde863038b77a1178

After this, I don't think making it a separate crate will be better.